### ISSUE TITLE
Headers don't work

### ISSUE THREAD

**Issue by victiondev** [FredyH/GWSockets#15]
My code for initializing a WS connection from gmod is

```lua
CLIENT = GWSockets.createWebSocket("ws://" .. Project.Globals.API.GetAPIBase() .. "/server/io", false)
CLIENT:setHeader("Authorization", Project.API.Key)

... -- omitted event handlers

CLIENT:open()
```

And for some reason when I log the headers on the receiving end of the connection, the Authorization header is not there...

Having a different header name changes nothing whether it is all lowercase, all uppercase. Any explanation or fix for this would be useful.

**Comment by FredyH** [comment]
The setHeader function returns a boolean indicating whether or not setting the header actually work, presumably it returns false for whatever reason, could you just make sure that is the case?

**Comment by victiondev** [comment]
Yea, it seems to work fine with 1.0.0, should I just use that for the time being?

**Comment by FredyH** [comment]
I fixed it in 1.1.2.
